sqlglot/generator.py

```diff
@@ -1130,7 +1135,13 @@ def expressions(self, expression, key=None, flat=False, indent=True, sep=", "):
         if flat:
             return sep.join(self.sql(e) for e in expressions)
 
-        expressions = self.sep(sep).join(self.sql(e) for e in expressions)
+        sql = [self.sql(e) for e in expressions]
+        if len(sql) > 1 and self.leading_comma:
+            remaining_expr = "\n".join([f"{sep}{s}" for s in sql[1:]])
+            expressions = f"  {sql[0]}\n{remaining_expr}"
```

Comment:
this doesn't seem to handle pretty vs non pretty? it always has a line break

Reply:
ah, good call, I'l add that in